src/core/mapping/instance_manager.cc

```diff
@@ -87,9 +87,6 @@ struct construct_overlapping_region_group_fn {
       size_t bound_vol = bound.volume();
       size_t union_vol = union_bbox.volume();
 
-      // If it didn't get any bigger then we can keep going
-      if (bound_vol == union_vol) continue;
```

Comment:
I already got this change in the other PR: https://github.com/nv-legate/legate.core/pull/350